deployments/infra/stacks/ami_pipeline_stack.go

```diff
@@ -288,7 +288,7 @@ phases:
               # Create .env file
               cat > .env << ENVEOF
               CHAIN_ID=$CHAIN_ID
-              DB_OWNER=postgres://kwild:kwild@kwil-postgres:5432/kwild
+              DB_OWNER=postgres://postgres@tn-postgres:5432/kwild
```

Comment:
Maybe I'm off on this, but db owner was the address that was about the address `DB_OWNER`

But if I'm not mistaken, we made those 2 env variables more explicit:

`SETUP_DB_OWNER` and `SETUP_CHAIN_ID`

The reason is that it is only used when it needs to generate a config on the first deployment

But right now these variables aren't being used anymore, right?

@coderabbitai can you check these?